### DISCUSSION TITLE
Strategy for analyzing second sample (alt sequencing platform) for one individual in a family

### DISCUSSION THREAD

**Discussion by jxchong** [broadinstitute/seqr#2273] [Q&A]
If I have short-read/Illumina WGS for a trio and then I do long read sequencing on the proband only, what would be the best way, if any, to leverage the new data in analysis in seqr? A similar scenario would be when I have exome data for the trio and Illumina WGS for the proband. In the case of long read data, we would not want to replace the old data for the proband because with current error rates for, e.g. Nanopore long read genomes, the Illumina WGS would be more accurate for most variants (but the long read data would have superior coverage in particular regions of the genome and for SVs).

Options:

1. Duplicate the project, substitute the new data for just the proband while keeping the parents on the older data, but that presumably would mess up using seqr for cross-project reporting and analysis and mean storing the parents' data twice? This is maybe? might not even be possible because my impression is seqr requires a multisample VCF and there's no method for making a multisample with long read variant calls for the proband and Illumina calls for the parents?
2. Add long read data another sample/individual in the existing project, so there would be essentially Proband 1 (Illumina) and Proband 2 (long read), and then you'd need to do custom analyses to specify which proband should be used in a given analysis. Or actually, is this even possible when we don't have a joint multisample call VCF with all 4 samples (parents + Proband 1 + Proband 2?
3. Make a new project with just the proband, but we wouldn't be able to, e.g. do a search for de novo variants since the parents would be in a separate project.
4. ??

**Comment by hanars** [comment]
We currently do not have any specific support for long range sequencing in seqr. I don't know much about it, but if the VCF format perfectly matches the VCF format for a standard GATK joint called VCF there is no reason why it wouldn't work in seqr. However, if thats not the case it would probably be about a month of FTE engineering work to build a loading pipeline that could correctly parse the long read data and export it in such a way that it would search and display properly in seqr. This is not work we have on our radar and realistically I don't think our team will have the bandwidth to add this support anytime soon. 

I don't think I understand the kinds of searches you want to do across different data types, but I think this is less a question about what the technology can do and more a question about what will actually produce meaningful biologically relevant results. If you take, say, exome data for the parents and combine it with genome data for a proband, this does not somehow enable you to do a meaningful inheritance based search. There will be thousands and thousands of variants in the proband that are not called at all in the parents because of the very nature of the data types. In these situations, what we do, and what is really the only thing you CAN do, is search in the exome trio data for results based on inheritance, and you will actually be able to filter out, say, true de novos from parentally inherited het variants. And then if you don't find anything interesting in the exome data you do a search in the proband only data in the genome, with all the limitations that entails. But to be clear, the presence of the parental exome data would in no way change what kind of searches you could do on that genome data. If you wish to discuss this further, I suggest you reach out to our analysis team rather than the seqr softawre engineering team, as like I said this isn't really a technology-related issue. 

That said, we do have some support for looking at exome and genome together. What we do in our seqr is have one project for our exome data  and another for the genome data. Mostly, we search in these projects separately, but you can search in them in the same search if you want to. To prevent reporting duplication we just make sure to only apply the actual discovery tag to whichever project we want to pull data from in the report, and use other tags in the other project. The other thing seqr supports is the ability to load multiple data types to a single project, providing they are on the same genome build. So you can add an exome index to a project, and then load a genome index to that same project. Search will then return variants found in either the exome or genome data. However, as I explained above, it will compute inheritance based only on a single data type. So if you apply an inheritance filter, for the exome data it will check the whole family and return exome varaints that match, and for the genome data it will check only the proband and return genome variants that match. The reason our group does not use this is they actually find it easier most of the time to keep the separate projects and explicitly search on the exome and genome data separately., rather than have seqr do the combined search.

**Reply by jxchong** [reply]
Thanks Hana, the long read data was more of an example, theoretical question. In discussions with Anne, we determined that supporting LR would be a good place for us to contribute, so no worries, I understand you're not planning to spend any effort there for the foreseeable future.

Your last paragraph is very helpful for us in figuring out what our options are, though!
